docs/migration_guides/qi_migration.rst

```diff
@@ -0,0 +1,738 @@
+
+        print(expectation_value)
+
+    .. code-block:: text
+
+        [-0.04101562]
+
+.. dropdown:: Example 3: Circuit Sampling on IBM Backend with Error Mitigation
+    :animate: fade-in-slide-down
+
+    **Using Quantum Instance**
+
+    The ``QuantumInstance`` interface allowed the configuration of measurement error mitigation settings such as the method, the
+    matrix refresh period or the mitigation pattern. This configuration is no longer available in the primitives
+    interface.
+
+    .. code-block:: python
+
+        from qiskit import QuantumCircuit
+        from qiskit.utils import QuantumInstance
+        from qiskit.utils.mitigation import CompleteMeasFitter
+        from qiskit_ibm_provider import IBMProvider
+
+        circuit = QuantumCircuit(2)
+        circuit.x(0)
+        circuit.x(1)
+        circuit.measure_all()
+
+        provider = IBMProvider()
+        backend = provider.get_backend("ibmq_montreal")
+
+        qi = QuantumInstance(
+            backend=backend,
+            shots=4000,
+            measurement_error_mitigation_cls=CompleteMeasFitter,
+            cals_matrix_refresh_period=0,
+        )
+
+        result = qi.execute(circuit).results[0].data
+        print(result)
+
+    .. code-block:: text
+
+        ExperimentResultData(counts={'11': 4000})
+
+
+    **Using Primitives**
+
+    The Runtime Primitives offer a suite of error mitigation methods that can be easily "turned on" with the
+    ``resilience_level`` option. These are, however, not configurable. The sampler's ``resilience_level=1``
+    is the closest alternative to the Quantum Instance's measurement error mitigation implementation, but this
+    is not a 1-1 replacement.
+
+    For more information on the error mitigation options in the Runtime Primitives, you can check out the following
+    `link <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/stubs/qiskit_ibm_runtime.options.Options.html#qiskit_ibm_runtime.options.Options>`_.
+
+    .. code-block:: python
+
+        from qiskit import QuantumCircuit
+        from qiskit_ibm_runtime import QiskitRuntimeService, Sampler, Options
+
+        circuit = QuantumCircuit(2)
+        circuit.x(0)
+        circuit.x(1)
+        circuit.measure_all()
+
+        service = QiskitRuntimeService(channel="ibm_quantum")
+        backend = service.backend("ibmq_montreal")
+
+        options = Options(resilience_level = 1) # 1 = measurement error mitigation
+        sampler = Sampler(session=backend, options=options)
+
+        # Run
+        result = sampler.run(circuit, shots=4000).result()
+        quasi_dists = result.quasi_dists
+
+        print("Quasi dists: ", quasi_dists)
+
+    .. code-block:: text
+
+        Quasi dists: [{2: 0.0008492371522941081, 3: 0.9968874384378738, 0: -0.0003921227905920063,
+		 1: 0.002655447200424097}]
+
+.. dropdown:: Example 4: Circuit Sampling with Custom Bound and Unbound Pass Managers
```

Comment:
The power of unbound and bound pass managers in the quantum instance is actually coming from the combination with the `CircuitSampler` 🙂 If you have an `unbound` and `bound` pass manager (for example, the unbound pass manager does the routing and high level optimization and the `bound` does pulse optimization), then you could do
```python
qi = QuantumInstance(pass_manager=unbound, bound_pass_manager=bound, ....)
sampler = CircuitSampler(qi)

circuits = .... # circuit with *free* parameters
sampler.convert(circuits, param_values=....)  # sampler will first do the unbound, then bind, then the bound for you
```
This workflow is (afaik) not possible anymore with the primitives. Should we update this example?